projects/ngrx.io/content/guide/entity/adapter.md

```diff
@@ -188,6 +189,9 @@ const userReducer = createReducer(
   on(UserActions.loadUsers, (state, { users }) => {
     return adapter.setAll(users, state);
   }),
+  on(UserActions.setUsers, (state, { users }) => {
```

Comment:
Could you also define the action in the `user.actions.ts` template please (right above this template)

Reply:
Applied at https://github.com/ngrx/platform/pull/3029/commits/4f298304a8af8e2bfce9f47628aec143a1822098